mne/time_frequency/tfr.py

```diff
@@ -333,10 +351,11 @@ def _time_frequency(X, Ws, use_fft):
         tfrs = _cwt_convolve(X, Ws, mode)
 
     for tfr in tfrs:
-        tfr_abs = np.abs(tfr)
-        psd += tfr_abs ** 2
-        plf += tfr / tfr_abs
-
+        psd += tfr
+        plf += tfr / np.abs(tfr)
+    psd = np.abs(psd) / n_epochs
+    psd *= psd
+    plf = np.abs(plf) / n_epochs
```

Comment:
Moved the absolute value, division, and squaring operations to this function to be more DRY (and avoid errors).